lib/autoupdate/start.rb

```diff
@@ -115,8 +115,20 @@ def start(interval:, args:)
       set_env << "\nexport SUDO_ASKPASS=#{env_sudo}"
     end
 
+    ac_only = if args.ac_only?
+      <<~EOS
+        if [[ $(pmset -g ps | head -1) =~ "Battery Power" ]]; then
+          echo "Not starting autoupdate, because device is running on Battery"
```

Comment:
```suggestion
        if [[ $(/usr/bin/pmset -g ps | /usr/bin/sed -nE "s/.*'(.*) Power.*/\1/p") != "AC" ]]; then
          echo "Skipping homebrew-autoupdate: device not on AC power" >&2
```

The `pmset|sed` will return one of: Battery, UPS, or AC. It seems like the intention here is to skip the autoupdate if not plugged into AC.